services/src/api/apidoc.rs

```diff
@@ -481,7 +481,7 @@ mod tests {
     use super::*;
     use crate::ge_context;
     use crate::util::tests::send_test_request;
-    use crate::{pro::contexts::PostgresContext, util::openapi_examples::can_run_examples};
+    use crate::{contexts::PostgresContext, util::openapi_examples::can_run_examples};
```

Comment:
unrelated to this line:
I suspect that removing the pro subfolder for test_data and providers_defs may lead to problems in 
- the dataset-defs scripts: there are scripts that assume providers (and data?) to be in a pro subfolder
- deployments: in the server-deployment are instances like the nightly that have pro providers configured

Reply:
- in deployments there are instances like the zentrale, which include pro providers. I am not sure if they still work after the change: https://github.com/geo-engine/server-deployment/blob/ac58adef67b9561f9f0bb1fa8d11d1a08acea725/peter/setup_pod_8350_geoengine_zentrale.sh#L21
- in the dataset-defs there is stuff like "skip_pro_providers" https://github.com/geo-engine/dataset-defs/blob/5084c5b8816693a5fe75b85082f7c0ef0f6f0ad1/deploy.py#L29
  - as they do no longer exist we should remove them there as well
  - we should check whether the deploy script still works for the provider that were once pro providers

Reply:
https://github.com/geo-engine/dataset-defs/pull/79

https://github.com/geo-engine/server-deployment/pull/67